docs/topics/con_springboot-features-and-frameworks-summary.adoc

```diff
@@ -5,7 +5,7 @@
 This guide covers the design of modern applications using link:https://projects.spring.io/spring-boot/[Spring Boot]. These concepts support developing Web or Websocket applications using either a HTTP connector or non-blocking HTTP connector.
```

Comment:
a HTTP--> an HTTP

Reply:
@snelluli1, thank you for this comment, but with this scope and time that we have left, I'd rather we focussed only on the changes Huntley is making in the pull request. There's definitely an opportunity to review the guides from a stylistic perspective because we have accumulated some debt over time, but that should happen in a separate issue. Thanks.

Reply:
Sure, @tradej. I will keep that in mind.